Makefile

```diff
@@ -23,11 +23,14 @@ gosec:
 server: $(SRCS)
 	go build ./cmd/server
 
+client: $(SRCS)
+	go build ./cmd/client
+
 test:
 	go test ./...
 
 clean:
-	rm -rf server
+	rm -rf server client
```

Comment:
tiny nit, may as well prefix the current path `./server` `./client` as someone may not have it in their $PATH (which is likely there fault, but it does no harm having them there).

Reply:
not following you - these paths mean the files named `client` and `server` in `$CWD`... even if you run make clean from a different dir, it only refers to files in the same directory as the Makefile